README.md

```diff
@@ -2,46 +2,157 @@
 
 ```
 sudo apt-add-repository ppa:ubuntugis/ubuntugis-unstable
 sudo apt-get update
 sudo apt-get install gdal-bin libgdal-dev
 ```
 
+Clone the Repo to your machine 
+
+``` git clone https://github.com/hotosm/galaxy-api.git ```
+
+Navigate to repo 
+
+``` cd galaxy-api ```
+
 Install python dependencies
 
 ```pip install -r requirements.txt```
 
-#### 2. Create ```config.txt``` inside src directory.
+Install gdal python ( Include your gdal version , if you are using different version ) 
+ 
+```pip install gdal==3.0.2```
+
+
+
+### 2. Create ```config.txt``` inside src directory.
+![image](https://user-images.githubusercontent.com/36752999/188402566-80dc9633-5d4e-479c-97dc-9e8a4999b385.png)
+
+
+### 3. Setup Underpass 
+  Run underpass from [here](https://github.com/hotosm/underpass/blob/master/doc/getting-started.md)  OR Create database "underpass" in your local postgres and insert sample dump from  ```/tests/src/fixtures/underpass.sql ```
+
+```psql -U postgres -h localhost underpass < underpass.sql```
+### 4. Setup Insights 
+Setup insights from [here](https://github.com/hotosm/insights) OR Create database "insights" in your local postgres and insert sample dump from  ```/tests/src/fixtures/insights.sql ```
+
+```psql -U postgres -h localhost insights < insights.sql```
+
+### 5. Setup Raw Data  
+Initialize rawdata from [here](https://github.com/hotosm/underpass/tree/master/raw) OR Create database "raw" in your local postgres and insert sample dump from  ```/tests/src/fixtures/raw_data.sql ```
+
+```psql -U postgres -h localhost raw < raw_data.sql```
+
+
+### 6. Setup Oauth 
+Login to [OSM](https://www.openstreetmap.org/) , Click on My Settings and register your local galaxy app to Oauth2applications
+
+![image](https://user-images.githubusercontent.com/36752999/188452619-aababf28-b685-4141-b381-9c25d0367b57.png)
+
+
+Check on read user preferences and Enter redirect URI as following
+```http://127.0.0.1:8000/latest/auth/callback/```
+
+Grab Client ID and Client Secret and put it inside config.txt as OAUTH Block , you can generate secret key for your application by yourself
+
 
-Following this [config sample](https://github.com/hotosm/galaxy-api/blob/master/src/config.txt.sample) setup your database configuration like this config block 
+### 7. Put your credentials inside config.txt
+Insert your config blocks with the database credentials where you have underpass ,insight and tm in your database
 
 ```
-[SOURCE_NAME]
+[INSIGHTS]
 host=localhost
+user=postgres
+password=admin
+database=insights
+port=5432
+
+[UNDERPASS]
+host=localhost
+user=postgres
+password=admin
+database=underpass
+port=5432
+
+[RAW_DATA]
+host=localhost
+user=postgres
+password=admin
+database=raw
+port=5432
+
+[OAUTH]
+client_id= your client id 
+client_secret= your client secret
+url=https://www.openstreetmap.org
+scope=read_prefs
+login_redirect_uri=http://127.0.0.1:8000/latest/auth/callback/
+secret_key=jnfdsjkfndsjkfnsdkjfnskfn
```

Comment:
The value of `secret_key` don't need to be changed ?

Reply:
I will advise to create your own from your preferred app or method . I have randomly typed that for sample , Advised on line 63